app/create_app.py

```diff
@@ -1,14 +1,23 @@
+import os
+
 from flask import Flask
 from flask_compress import Compress
 from flask_talisman import Talisman
 
 
-def create_app() -> Flask:
+def create_app(testing=False) -> Flask:
 
     # ---- SETUP STATIC URL PATH.
     flask_app = Flask(__name__)
 
-    flask_app.config.from_pyfile("config.py")
+    # If testing or in dev print current config
+    if (os.environ.get("FLASK_ENV") == "development") | testing:
+        flask_app.config.from_object("config.development.DevelopmentConfig")
+        from config.development import DevelopmentConfig
+
+        DevelopmentConfig.pretty_print()
+    else:
+        flask_app.config.from_object("config.default.DefaultConfig")
```

Comment:
This explicitly sets the default config in all environments. If we're implementing a switchable environment config convention across repos do we need to implement that here? [This method](https://github.com/communitiesuk/funding-service-design-authenticator/blob/0beecf1bc635dc2ab331d557cc5efa44f4d3b54a/app.py#L42) could be used, which switches based on FLASK_ENV [in the config __init__](https://github.com/communitiesuk/funding-service-design-authenticator/blob/main/config/__init__.py) with individual environment class files in an environments sub folder